File: android/EyeTimeTrackerAndroid/src/com/eyetimetracker/android/TodayTone.java
package com.eyetimetracker.android;

public enum TodayTone {
    SAFE,
    WARN,
    DANGER;

    private static final long SAFE_SECONDS = 6L * 3600L;
    private static final long WARN_SECONDS = 8L * 3600L;

    public static TodayTone fromSeconds(long totalSeconds) {
        long safeSeconds = Math.max(0L, totalSeconds);
        if (safeSeconds <= SAFE_SECONDS) {
            return SAFE;
        }
        if (safeSeconds <= WARN_SECONDS) {
            return WARN;
        }
        return DANGER;
    }
}

File: src/EyeTimeTracker.Core/Reminders/TodayTone.cs
namespace EyeTimeTracker.Core.Reminders;

public enum TodayTone
{
    Safe,
    Warn,
    Danger
}

public static class TodayTonePolicy
{
    private const long SafeSeconds = 6L * 3600L;
    private const long WarnSeconds = 8L * 3600L;

    public static TodayTone FromSeconds(long totalSeconds)
    {
        var safeSeconds = Math.Max(0, totalSeconds);
        if (safeSeconds <= SafeSeconds)
        {
            return TodayTone.Safe;
        }

        return safeSeconds <= WarnSeconds ? TodayTone.Warn : TodayTone.Danger;
    }
}
